src/XrdHttp/XrdHttpProtocol.hh

```diff
@@ -436,6 +474,10 @@ protected:
   /// Rules that turn HTTP headers to cgi tokens in the URL, for internal comsumption
   static std::map< std::string, std::string > hdr2cgimap;
 
+  /// Rules that turn HTTP headers to cgi tokens in the URL; if the header is given multiple times, they
+  /// will be added to the CGI in a comma-separate manner.
+  static detail::HeaderMap hdr2cgimultimap;
```

Comment:
Are these `static` variables declared in a header correct? These look like they should have been declared as `static inline` to not have a copy in each translation unit.

Reply:
Is that true?  From https://en.cppreference.com/w/cpp/language/static, it states that statics are external linkage (meaning you have to explicitly initialize them in one of the translation units, which is done in `XrdHttpProtocol.cc`).

I'm not an expert in this area of C++ but copying initializing a member into each translation unit that imports a header would be quite different from my understanding of statics.  If you have any links for me to read -- since I see `static inline` was introduced in C++17 -- I'd appreciate it!